db/migrate/20150330093200_alter_visualization_id_type_uuid.rb

```
@@ -0,0 +1,30 @@
+Sequel.migration do
+
+  up do
+    Rails::Sequel.connection.run(%Q{ ALTER TABLE external_sources drop constraint if exists external_sources_visualization_id_fkey; })
+
+    begin
+      Rails::Sequel.connection.run(%Q{ ALTER TABLE external_sources ALTER COLUMN visualization_id TYPE uuid USING (visualization_id::uuid); })
+      Rails::Sequel.connection.run(%Q{ ALTER TABLE visualizations ALTER COLUMN id TYPE uuid USING (id::uuid); })
+    rescue => e
+      puts "There was an error converting visualizations.id (or external_sources.visualization.id) from type text to uuid, please check id format: #{e.inspect}"
```

Sure, my bad. It should raise the exception. I think raising it again is the best option.